clinlp/entity.py

```diff
@@ -213,3 +214,40 @@ def __call__(self, doc: Doc):
         doc.set_ents(entities=ents)
 
         return doc
+
+
+def create_term(row: pd.Series, col_term: str) -> Term:
```

Comment:
Mooier zou eigenlijk nog zijn om een dict te geven als input in plaats van een Series, omdat dat helemaal python native is

Reply:
Dit is een helper functie die op een Dataframe wordt toegepast via .apply(). Het eerste argument is dan altijd een pandas Series; dat is hoe die apply functie werkt volgens de documentatie. Dus dan kan wat je suggereert niet.